app/src/main/java/id/rizmaulana/covid19/data/model/CovidDaily.kt

```diff
@@ -12,10 +12,10 @@ data class CovidDaily(
     @Expose @SerializedName("deltaConfirmed") val deltaConfirmed: Int = 0,
     @Expose @SerializedName("deltaRecovered") val deltaRecovered: Int = 0,
     @Expose @SerializedName("mainlandChina") val mainlandChina: Int = 0,
-    @Expose @SerializedName("objectid") val objectid: Int = 0,
+    @Expose @SerializedName("objectId") val objectId: Int = 0,
     @Expose @SerializedName("otherLocations") val otherLocations: Int = 0,
     @Expose @SerializedName("reportDate") val reportDate: Long = 0,
-    @Expose @SerializedName("reportDateString") val reportDateString: String? = null,
+    @Expose @SerializedName("reportDateString") val reportDateString: String = "",
```

Comment:
Afaik, ini kalau data dari API-nya `null` string tetap jadi `null` mas @isfaaghyth 
Reference : https://proandroiddev.com/safe-parsing-kotlin-data-classes-with-gson-4d560fe3cdd2

Kalau benar2 perlu gak nullable, supaya safe biasanya pakai custom getter 

```suggestion
    @Expose @SerializedName("reportDateString") private val _reportDateString: String? = null,
    ...
    val reportDateString get() = _reportDateString.orEmpty()
```

Reply:
ok noted!